lib/puppet/functions/ldapquery/query.rb

```diff
@@ -16,6 +16,7 @@
         Optional[server] => String[1],
```

Comment:
This is a single server .

The module supports multiple servers as 

```
ldap.hosts = [[ldap1.ex.org,389], [ldap2.ex.org, 389]]
```
if reworking would be good.

Reply:
I'm going to work on the docs and examples this morning.  Hopefully it'll become clearer.